alfresco-transform-aio/alfresco-transform-aio/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>alfresco-transform-aio</artifactId>
+    <name>Alfresco All In One Transformer</name>
```

Comment:
Minor: wondering if we should reference the fact that it is an "all-in-one" transformer for *core* transformers ?

Hence, call it Alfresco Core (All-in-One") Transformer or similar ? We may also wish to do the same for the docker image (and later zip file), eg. alfresco-transform-core-aio

Reply:
Lets raise this as a separate task